department-of-veterans-affairs/va.gov-cms
# VACMS-9290: fix ordering issue with Billing and Insurance copay fields

## Description

Closes #9290 

## Testing done

Visual testing

## Screenshots

![Screen Shot 2022-06-02 at 12 33 02 PM](https://user-images.githubusercontent.com/21045418/171684768-15ba1e1e-8839-4cf0-87cd-414009283da1.png)

## QA steps

As an admin user:
1. Visit /admin/content and filter by content type = VAMC System Billing and Insurance
2. Open a handful of nodes in both node view and node edit
3. Verify the order matches what is displayed in the above screenshot

### Select Team for PR review
